susemanager/susemanager.changes.mbussolotto.bootstrap_data_persist

```diff
@@ -0,0 +1,2 @@
+- Mgr-create-bootstrap-repo needs to import modules from a
```

Comment:
You need to align the changelog entries between the two ports or it will be a mess later on.

Reply:
I created this PR https://github.com/SUSE/spacewalk/pull/27299 . Changelog requirements for master are more restrictive, so I need to change the other one